functional-test/src/test/groovy/org/rundeck/tests/functional/api/scm/ScmPluginActionsSpec.groovy

```diff
@@ -0,0 +1,174 @@
+package org.rundeck.tests.functional.api.scm
+
+import org.rundeck.tests.functional.api.ResponseModels.Job
+import org.rundeck.util.annotations.APITest
+import org.rundeck.util.api.scm.GitScmApiClient
+import org.rundeck.util.api.scm.gitea.GiteaApiRemoteRepo
+import org.rundeck.util.api.scm.httpbody.IntegrationStatusResponse
+import org.rundeck.util.api.scm.httpbody.ScmActionInputFieldsResponse
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.rundeck.util.api.scm.httpbody.GitExportSetupRequest
+import org.rundeck.util.api.scm.httpbody.ScmActionPerformRequest
+import org.rundeck.util.api.scm.httpbody.SetupIntegrationResponse
+import org.rundeck.util.api.storage.KeyStorageApiClient
+import org.rundeck.util.container.BaseContainer
+
+@APITest
+class ScmPluginActionsSpec extends BaseContainer {
+    static final String PROJECT_NAME = 'ScmPluginActionsSpec'
+    static final String BASE_EXPORT_PROJECT_LOCATION = '/projects-import/scm/project-scm-export-one-job.rdproject.jar'
+    static final GiteaApiRemoteRepo remoteRepo = new GiteaApiRemoteRepo('repoExample')
+
+    def setupSpec() {
+        remoteRepo.setupRepo()
+    }
+
+    def "project scm status must be export needed when having a new job"(){
+        given:
+        String projectName = "${PROJECT_NAME}-P1"
+        setupProject(projectName, BASE_EXPORT_PROJECT_LOCATION)
+        GitScmApiClient scmClient = new GitScmApiClient(clientProvider).forIntegration(integration).forProject(projectName)
+
+        expect:
+        scmClient.callSetupIntegration(GitExportSetupRequest.defaultRequest().forProject(projectName).withRepo(remoteRepo)).response?.success
+
+        when:
+        IntegrationStatusResponse retrievedStatus = scmClient.callGetIntegrationStatus().response
+
+        then:
+        verifyAll {
+            retrievedStatus.integration == integration
+            retrievedStatus.message == 'Some changes have not been committed'
+            retrievedStatus.synchState == 'EXPORT_NEEDED'
+            retrievedStatus.project == projectName
+            retrievedStatus.actions.size() == 1
+            retrievedStatus.actions.first() == 'project-commit'
+        }
+
+        where:
```

Comment:
there is no point on having the integration in "where" block, since it's only 1 integration